RATapi/utils/plotting.py

```diff
@@ -105,68 +339,55 @@ def plot_ref_sld_helper(
     ref_plot.cla()
     sld_plot.cla()
 
-    for i, (r, sd, sld, name) in enumerate(
-        zip(data.reflectivity, data.shiftedData, data.sldProfiles, data.contrastNames),
-    ):
-        # Calculate the divisor
-        div = 1 if i == 0 and not q4 else 2 ** (4 * (i + 1))
-        q4_data = 1 if not q4 or not data.dataPresent[i] else sd[:, 0] ** 4
-        mult = q4_data / div
-
-        # Plot the reflectivity on plot (1,1)
-        ref_plot.plot(r[:, 0], r[:, 1] * mult, label=name, linewidth=2)
+    plot_data = _extract_plot_data(data, q4, show_error_bar)
+    for i, name in enumerate(data.contrastNames):
+        ref_plot.plot(plot_data["ref"][i][0], plot_data["ref"][i][1], label=name, linewidth=1, animated=animated)
```

Comment:
The linewidth has been narrowed here for the reflectivity plots (was previously 2). Was that a deliberate choice?

Reply:
The line widths are 1 for the other lines so changed for consistency